### ISSUE TITLE
project doesn't generate with workbox 

### ISSUE THREAD

**Issue by husayt** [nuxt-community/modules#101]
I get this error on npm run generate

```
Error: workbox-sw is needed by the service worker and could not be copied over to your new site. Error: EPERM: operation not permitted, open 'c:\zzz\nuxt\app\static\workbox-sw.prod.v1.1.0.js.map'
    at Promise.all.then.catch (c:\zzz\nuxt\node_modules\workbox-build\build\lib\utils\copy-workbox-sw.js:17:13)
    at <anonymous>
```

**Comment by pi0** [comment]
Hi. This issue is sometimes happening with workbox during generate and is not by us. Currently, you may simply run it again and it works. I'll investigate more finding root cause.

**Comment by phillipcurl** [comment]
Hi, I'm also seeing this when running `nuxt build` in addition to `nuxt generate`. As you said @pi0, re-running the command seems to do the trick, which is great when working locally or generating and hosting the static site somewhere. Unfortunately, I'm deploying a Nuxt app to Now and am running the build step at deploy-time, so it's causing the deployment to fail. Any chance you have some more details on this, or a link to the issue on workbox? I'm happy to look into this more and provide a fix if possible, as it's a bit of a critical issue for me and I've loved working with Nuxt and the PWA plugin. Thanks!

**Comment by pi0** [comment]
This should be fixed with `@nuxtjs/workbox:^1.1.5` please upgrade :)
